libethcore/Precompiled.cpp

```diff
@@ -132,15 +132,34 @@ ETH_REGISTER_PRECOMPILED(modexp)(bytesConstRef _in)
 	return {true, ret};
 }
 
+namespace
+{
+	bigint expLengthAdjust(bigint _baseLength, bigint _expLength, bytesConstRef _in)
+	{
+		size_t const numConsideredBytes(min(_expLength, {32}));
```

Comment:
Yeah, cost change affects state, so you definitely need to regenerate state tests for Metropolis